### ISSUE TITLE
Compute SBERT broken with imported valid/test

### ISSUE THREAD

**Issue by jboelaert** [emilienschultz/pyactivetigger#1043]
After importing an external valid/test set, SBERT computation seems broken. After reaching 100%, I get "Error in feature : Features don't have the right shape".

Problem seems general, I've tried with several datasets and embedding models, and embedding works when the same datasets are imported as training sets.

**Comment by emilienschultz** [comment]
I look at it

**Comment by emilienschultz** [comment]
ok, I pushed a fix, could you have a look if it works now ?

**Comment by jboelaert** [comment]
Working on new projects, thanks!

**Comment by jboelaert** [comment]
Also working on old projects!
